pom.xml

```diff
@@ -58,6 +58,11 @@
         </dependencies>
     </dependencyManagement>
     <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>version-number</artifactId>
+            <version>1.10</version>
+        </dependency>
```

Comment:
This is already included in Jenkins core as of 2.342, so rather than duplicate the library in this plugin, I suggest bumping the core baseline to 2.346 or later.

Reply:
Something like deleting this new dependency and instead doing:

```
diff --git a/pom.xml b/pom.xml
index 22f57f5..573bf80 100644
--- a/pom.xml
+++ b/pom.xml
@@ -51,15 +51,15 @@
     <properties>
         <revision>2.3</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.277.4</jenkins.version>
+        <jenkins.version>2.346.1</jenkins.version>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     </properties>
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.277.x</artifactId>
-                <version>984.vb5eaac999a7e</version>
+                <artifactId>bom-2.346.x</artifactId>
+                <version>1382.v7d694476f340</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
```

Reply:
Implemented in 503ada6